lib/phoenix/code_reloader/colors.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Phoenix.CodeReloader.Colors do
+  defp css_color(n) when is_binary(n),
+    do: String.to_integer(n) |> css_color()
+  defp css_color(31), do: :red
+  defp css_color(33), do: "#b3b30a"
```

Comment:
I’ve rewritten the Colors module to handle more complex inputs (combined/overwritten styles etc.) and added some tests.